libs/megadrive-graphics/src/lib.rs

```diff
@@ -1,8 +1,12 @@
 #![no_std]
 
 use core::mem::MaybeUninit;
+
 use megadrive_sys::vdp::{Sprite, VDP};
 
+pub mod default_ascii;
```

Comment:
I think this also needs to be changed to make it build again.

```suggestion
pub mod default_ascii {
    include!(concat!(env!("OUT_DIR"), "/default_ascii.rs")))
}
```

Reply:
Ah I was just wondering how that should work

Reply:
Oh, but now it doesn't compile, let me check

Reply:
The Docker image now requires Python of course, fixed in d1a4cacc2421c98c9004b7d6e9098ab899b4e256